src/group_theory/perm/subgroup.lean

```diff
@@ -12,31 +12,50 @@ import group_theory.subgroup
 This file provides extra lemmas about some `subgroup`s that exist within `equiv.perm α`.
 `group_theory.subgroup` depends on `group_theory.perm.basic`, so these need to be in a separate
 file.
+
+It also provides decidable instances on membership in these subgroups, since
+`monoid_hom.decidable_mem_range` cannot be inferred without the help of a lambda.
```

Comment:
[Docs for `monoid_hom.decidable_mem_range`](https://leanprover-community.github.io/mathlib_docs/group_theory/subgroup.html#monoid_hom.decidable_mem_range) for reference